src/libraries/Directory.Build.props

```diff
@@ -328,4 +328,9 @@
     <CLSCompliant Condition="'$(CLSCompliant)' == '' and ('$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true')">false</CLSCompliant>
     <CLSCompliant Condition="'$(CLSCompliant)' == ''">true</CLSCompliant>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <LibrariesTrimmedArtifactsPath>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'ILLinkTrimAssembly', '$(BuildSettings)', 'trimmed-runtimepack'))</LibrariesTrimmedArtifactsPath>
```

Comment:
Does this property have to go into the common props? It is only used in one place in `illink-sharedframework.targets`. Can it just be moved to `illink-sharedframework.targets`, so it is closer to where it is used, and it doesn't clutter the other projects with unnecessary properties?